design/open/uncomitted-gc.md

```diff
@@ -161,4 +161,13 @@ Optimized run uses the previous GC run output, to perform a partial scan of the
 * Since this solution relies on the new repository structure, it is not backwards compatible. Therefore, another solution will be required for existing 
 repositories
 * Even with the given optimizations, the GC process is still very much dependent on the amount of changes that were made on the repository
-since the last GC run.
\ No newline at end of file
+since the last GC run.
+
+## Performance Requirements
```

Comment:
I'm missing the actual requirement below. It is sort of an estimation of a single task of the uncommitted GC. How long will the entire GC run? What's the repo status (like number of commits, branches, objects to delete, etc.)

Reply:
Done